docs/user-guide/logs/write-logs.md

```diff
@@ -102,6 +102,67 @@ processors:
 
 It is recommended to use `dissect` or `regex` processor to split the input line into fields first and then process the fields accordingly.
 
+## Build-in Pipeline
+
+Currently we only have a built-in pipeline called `greptime_identity` that stores user input as is. We'll add an additional `greptime_timestamp` to the user data to indicate when it was written. We'll convert the json data into our table structure, and our conversion follows a few rules
+
+1. We will scan all the records in a batch of data and get a superset of the list of fields to be considered as the table schema of the batch. please be careful to keep the data fields fixed as much as possible.
+2. An error is returned when the same field has a different type on a different line.
+3. We ignore all fields with null value.
+
+### Type conversion rules
```

Comment:
```suggestion
## Built-in Pipelines

GreptimeDB offers built-in pipelines for common log formats, allowing you to use them directly without creating new pipelines.

### `greptime_identity`

The `greptime_identity` pipeline is designed for writing JSON logs and automatically creates columns for each field in the JSON log.

- The first-level keys in the JSON log are used as column names.
- An error is returned if the same field has different types.
- Fields with `null` values are ignored.
- An additional column, `greptime_timestamp`, is added to the table as the time index to indicate when the log was written.

#### Type conversion rules
```

Reply:
please move this section to `manage-pipelines.md`, the location would be before `## Create a Pipeline`